work_queue/src/work_queue_worker.c

```diff
@@ -618,6 +618,13 @@ static int handle_tasks(struct link *master)
 				debug(D_WQ, "task %d (pid %d) exited abnormally with signal %d",p->task->taskid,p->pid,p->exit_status);
 			} else {
 				p->exit_status = WEXITSTATUS(status);
+				FILE *loop_full_check;
+				if(p->loop_mount == 1 && (loop_full_check = fopen("./disk_alloc_report.txt", "r"))) {
+					p->task_status = WORK_QUEUE_RESULT_RESOURCE_EXHAUSTION;
+					p->task->disk_alloc_full = 1;
+					fclose(loop_full_check);
+					unlink("./disk_alloc_report.txt");
```

Comment:
I don't like that this name is fixed, as it may collide with a user file.  A better way would be to create a tmp file and pass the name to your library, perhaps with an environment variable.